### PR TITLE
[Improve] support head loss or postprocessor is None for only infer

### PR DESCRIPTION
**Pull request by Harold-lkk**

### CODE REVIEW

File: mmocr/models/textdet/heads/base.py [hunk 1]
```diff
@@ -53,15 +53,20 @@ class BaseTextDetHead(BaseModule):
...
+                 module_loss: Optional[Dict],
+                 postprocessor: Optional[Dict],
                  init_cfg: Optional[Union[Dict, List[Dict]]] = None) -> None:
         super().__init__(init_cfg=init_cfg)
-        assert isinstance(module_loss, dict)
-        assert isinstance(postprocessor, dict)
-
-        self.module_loss = MODELS.build(module_loss)
-        self.postprocessor = MODELS.build(postprocessor)
+        if module_loss is not None:
```
Comment (gaotongxiao) [inline mmocr/models/textdet/heads/base.py:60]:
```suggestion
        if getattr(self, 'module_loss', None) is not None:
```
Otherwise this line will result in undefined error when nothing is assigned to `module_loss`. Same for postprocessor

**Review by gaotongxiao — APPROVED** [review]
